packages/navigator/src/browser/navigator-frontend-module.ts

```diff
@@ -21,18 +20,19 @@ import { FuzzySearch } from './fuzzy-search';
 import { FileNavigatorSearch } from './navigator-search';
 import { SearchBox, SearchBoxProps, SearchBoxFactory } from './search-box';
 import { SearchBoxDebounce, SearchBoxDebounceOptions } from './search-box-debounce';
-
 import '../../src/browser/style/index.css';
+import { MenuContribution } from '@theia/core/lib/common/menu';
 
 export default new ContainerModule(bind => {
     bindFileNavigatorPreferences(bind);
     bind(FileNavigatorFilter).toSelf().inSingletonScope();
     bind(FileNavigatorContribution).toSelf().inSingletonScope();
```

Comment:
fyi: you can bind view contribution with one liner:  `bindViewContribution(bind, FileNavigatorContribution);`

It will register all contributions under the hood.